modules/mocked_authentication/lib/mocked_credential/service.rb

```diff
@@ -0,0 +1,59 @@
+
+    module MockedAuthentication
+      module MockedCredential
```

Comment:
Should we name this `Vamock` to match `Logingov` and `Idme`
```suggestion
      module Vamock
```